functors_task_1/student/include/impl.h

```diff
@@ -16,18 +25,23 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(const BeerBrand& beer)
 {
+    std::set<BeerBrand> expensiveBeers = {BeerBrand::Corona, BeerBrand::HoeGaarden};
```

Comment:
1. Why do you need to recreate it each function call?
2. Why do you need ordered container?
3. Please keep it simple